src/css/components/_adaptive.scss

```diff
@@ -34,7 +34,7 @@
 .video-js.vjs-layout-small:not(.vjs-fullscreen) {
   .vjs-current-time, .vjs-time-divider, .vjs-duration, .vjs-remaining-time,
   .vjs-playback-rate,
-  .vjs-mute-control, .vjs-volume-control,
+  .vjs-mute-control, .vjs-volume-panel,
```

Comment:
according to the comment above, `vjs-volume-panel` shouldn't be here because it's a replacement for the volume menu button.